app/views/pages/privacypolicy.html.erb

```diff
@@ -0,0 +1,223 @@
+<% provide(:title, 'Privacy Policy') %>
+
+<% cache do %>
+<div class="container">
+  <h1 class=about_h1>The Archives Unleashed Privacy Policy </h1>
+    <p class="about_p">We recognize the importance of and are committed to protecting the privacy of all users. The following privacy policy guides and outlines the Archives Unleashed Project’s online information practices.</p>
```

Comment:
`... the Archives Unleashed Project's (AU) online information practices.`

We use `AU` as a general acronym later so best to define it here.